components/Header.vue

```diff
@@ -1,10 +1,25 @@
+<style lang="scss">
+h1 {
+  margin-top: -10px;
+
+  a {
+    padding-left: 80px;
+    color: #3B75B9;
+    background: no-repeat url("/teh-128px.png") left;
+    background-size: 64px 64px;
+    display: inline-block;
+    height: 64px;
+    line-height: 76px;
+  }
+}
+</style>
+
 <template>
   <header>
-  <h1>
-    <NuxtLink v-if="$nuxt.$route.name != 'index'" to="/">The Thousand Ether Homepage</NuxtLink>
-    <span v-else>The Thousand Ether Homepage</span>
```

Comment:
so its always clickable? I like this i think

Reply:
Yea should always be clickable, some people obsess about this lol.